src/main/java/uk/gov/hmcts/reform/civil/service/SolicitorEmailValidation.java

```diff
@@ -77,8 +76,7 @@ && checkIfOrgIdExists(civilCaseData.getRespondent1OrganisationPolicy())) {
             }
         }
 
-        if (YES.equals(gaCaseData.getIsMultiParty())
-            && NO.equals(civilCaseData.getRespondent2SameLegalRepresentative())) {
+        if (YES.equals(gaCaseData.getIsMultiParty())) {
```

Comment:
why are you removing this condition? I believe it was the fix for functional test failure. Did you run functional test for your change?

Reply:
Good spot - I am fixing it.

Reply:
Added back - build in green.